pom.xml

```diff
@@ -63,6 +63,8 @@
                 <configuration>
                     <parallel>all</parallel> <!-- Run tests in parallel-->
                     <useUnlimitedThreads>true</useUnlimitedThreads>
+                    <argLine>-Xms1g</argLine>
+                    <argLine>-Xmx1g</argLine>
```

Comment:
Ensures unit tests have enough memory to run, especially in parallel.
A BWAPI frame takes 31MB, and thus asynchronous tests require N * 31MB free contiguous memory blocks to run.
The synchronization test environment will check that sufficient memory is available at runtime.